### DISCUSSION TITLE
Autocomplete Docker with Gem

### DISCUSSION THREAD

**Discussion by ariefsn** [ohmyzsh/ohmyzsh#12725] [Help] [answered]
Hi,
When I use docker and gem plugins, the docker suggestions do not work.
If I type `docker` and `tab`, it shows the `gem`'s suggestions instead.
Any idea?

Thanks 🙏🏻

**Comment by carlosala** [comment, marked as answer]
Thanks for the report, it was a typo! Fixed on the latest version, `omz update` should fix it!
